ocdaction/core/views.py

```diff
@@ -24,12 +28,19 @@ class AboutView(TemplateView):
     template_name = "core/about.html"
 
 
-class ActView(TemplateView):
+@login_required
+def ActView(request):
     """
     The Act View.
     """
     template_name = "core/act.html"
 
+    # Get the logged in user, then retrieve the users unarchived tasks
+    current_user = OCDActionUser.objects.get(id=request.user.id)
```

Comment:
There is a way to get the user from the current request and then completely remove this line, you can use `instance=request.user` and then use it straight on the next line. This will get the currently authenticated user